libs/template/presets/src/features/fulfillment/app.ts

```diff
@@ -11,32 +12,34 @@ import {
   FulfillmentRootFeatureConfig,
 } from './feature';
 
-export const fulfillmentResources: Resources = {
-  graphics: {
-    ...resourceGraphics,
-  },
-};
-
-export interface FulfillmentFeaturesConfig {
-  fulfillmentRoot?: FulfillmentRootFeatureConfig;
-  picking?: PickingFeatureConfig;
-}
-
 export function fulfillmentFeatures(
   config?: FulfillmentFeaturesConfig
 ): AppFeature[] {
   return [
     uiFeature,
     cartFeature,
     coreFeature,
-    {
-      resources: fulfillmentResources,
-    },
-    new WebPushNotificationFeature(),
     new RouterFeature(),
+    new I18nFeature(config?.i18n ?? defaultFulfillmentI18nConfig),
+    new WebPushNotificationFeature(),
+    { resources: fulfillmentResources },
     new FulfillmentRootFeature(config?.fulfillmentRoot),
     new PickingFeature(config?.picking),
   ];
 }
 
+export interface FulfillmentFeaturesConfig {
+  fulfillmentRoot?: FulfillmentRootFeatureConfig;
+  picking?: PickingFeatureConfig;
+  i18n?: I18nFeatureOptions;
+}
+
 export const fulfillmentTheme = { ...theme };
+
+export const fulfillmentResources: Resources = {
+  graphics: { ...resourceGraphics },
+};
+
+export const defaultFulfillmentI18nConfig: I18nFeatureOptions = {
+  locale: { defaultLocale: 'en' },
```

Comment:
what about moving `en` as the default in to the `LocaleService`?

Reply:
Actually now that we've made 'en' a fallback locale I will remove this config altogether.